### ISSUE TITLE
Artwork not showing up on Universal Media Player

### ISSUE THREAD

**Issue by DarkWolf7** [kalkih/mini-media-player#410]
Running Home Assistant 0.114.4

edited to add some details.

Built a media player component today that selects from my ShieldTV as a cast-based media player, as an Android TV-based media_player, and my PS4. However, despite the fact that the universal player populates artwork just fine, the mini player does not show it:

![image](https://user-images.githubusercontent.com/16918633/92314869-c44c4400-efab-11ea-8e55-b40fe4609fdb.png)

For what it's worth, I am using a state-template to overwrite the state - entirely because depending on what app I'm using, the artwork/metadata fills out better on one vs the other method of accessing the Shield. I don't know why this is.

I did a close inspection on the code on the page and it doesn't seem to be rendering anything at all for the background-image in the web code:

```
          <div class="mmp-player__core flex">
            
      <div class="entity__icon">
        <ha-icon></ha-icon>
      </div>
            <div class="entity__info">
              
      <div class="entity__info__name">
        Shield Media Player 
      </div>
              
      <div class="entity__info__media" style="animation-duration: falses;">
        
        <!----><span class="attr__media_title"><!---->Aladdin<!----></span><!----><span class="attr__app_name"><!---->VUDU<!----></span><!---->
      </div>
            </div>
            <mmp-powerstrip>
            </mmp-powerstrip>
          </div>
          <div class="mmp-player__adds">
            
              <mmp-media-controls>
              </mmp-media-controls>
            
            <mmp-shortcuts>
            </mmp-shortcuts>
            
            <mmp-group-list>
            </mmp-group-list>
          </div>
```
vs mini-media-player when pointing at the shield's media_player directly:
![image](https://user-images.githubusercontent.com/16918633/92314947-d7134880-efac-11ea-8115-b630608b624d.png)

Universal player config:
```
  - platform: universal
    name: Shield Media Player
    state_template: >
            {% if is_state('media_player.playstation_4','Playing') %}
            {{ states('media_player.playstation_4') }}
            {% elif is_state_attr('media_player.shieldtv', 'source','SageTV Miniclient') %}
            {{ states('media_player.shieldtv') }}
            {% else %}
            {{ states('media_player.shield') }}
            {% endif %}
    children:
      - media_player.shield
      - media_player.shieldtv
      - media_player.playstation_4
    commands:
      turn_on:
        service: remote.turn_on
        entity_id: remote.living_room
      turn_off:
        service: remote.turn_off
        entity_id: remote.living_room
      volume_up:
        service: remote.send_command
        entity_id: rremote.living_room
        data:
          device: B & K AV Receiver
          command: VolumeUp
      volume_down:
        service: remote.send_command
        entity_id: remote.living_room
        data:
          device: B & K AV Receiver
          command: VolumeDown
```
State attributes:
```
volume_level: 1
is_volume_muted: false
media_content_id: Vudu
media_duration: 7678.25
media_position: 2067.314
media_position_updated_at: '2020-09-05T23:16:29.493170+00:00'
media_title: Aladdin
app_id: AndroidNativeApp
app_name: VUDU
active_child: media_player.shield
friendly_name: Shield Media Player
entity_picture: 'http://images2.vudu.com/poster2/1121066-338'
supported_features: 22463
```

**Comment by agneevX** [comment]
Please share your card config.

**Comment by earion68** [comment]
Same here, similar config ant the card config is straight forward

```
type: 'custom:mini-media-player'
entity: media_player.universal
artwork: cover
```

State attributes:
```
volume_level: 0.5
is_volume_muted: false
media_content_id: U0XcqF7rqHk
media_content_type: music
media_duration: 321
media_position: 0.747
media_position_updated_at: '2020-12-18T19:37:08.814950+00:00'
media_title: 21 Guns
media_artist: Green Day
media_album_name: 21st Century Breakdown
app_id: 2DB7CC49
app_name: YouTube Music
active_child: media_player.maison
friendly_name: Universal
entity_picture: 'https://i.ytimg.com/vi/U0XcqF7rqHk/hqdefault.jpg'
supported_features: 152511

```

**Comment by kalkih** [comment]
Possibly solved by #449

**Comment by kalkih** [comment]
#449 fix is now merged and is in the latest release, let me know if not resolved.
